shaders/Maths/VectorAdd.osl

```diff
@@ -38,7 +38,7 @@ shader VectorAdd
 (
 	vector a = 0,
 	vector b = 0,
-	output vector out = 0
+	output vector out = 0 [[ string correspondingInput = "a" ]]
```

Comment:
Let's add a test case asserting that we get the expected shader network when this shader is disabled, and likewise that the pass-through connection is made when it is deleted.